config/providers.yml

```diff
@@ -3,17 +3,17 @@
     provider_display_name: Hetzner,
     provider_internal_name: hetzner,
     locations: [
-      {display_name: eu-central-h1, internal_name: hetzner-fsn1, visible: true },
-      {display_name: eu-north-h1, internal_name: hetzner-hel1, visible: true },
-      {display_name: github-runners, internal_name: github-runners, visible: false }
+      {display_name: eu-central-h1, internal_name: hetzner-fsn1, ui_name: Germany, visible: true },
```

Comment:
Our regions currently have 3 different names: `internal_name`, `display_name`, and `ui_name`. 🤯  I wish to combine `internal` and `display` at some point